src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import PeopleTable from './PeopleTable';
+import ErrorMessage from './ErrorMessage';
+import NotPeople from './NotPeople';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
```

Comment:
When catching an error, it's usually helpful to log the error or handle it in a way that can be used for debugging or user feedback. Consider adding an error parameter to the catch block and either logging it or using it to display a more specific error message.